### PR TITLE
Bump AG to Ubuntu 22.04

### PR DESCRIPTION
**Pull request by japplefield**
Updates to Ubuntu 22.04 on the AG. Requires us to setup the AG for P1 again @MattyMay

### CODE REVIEW

**Review by awdeorio — APPROVED** [review]
LGTM

**Comment by japplefield** [comment]
Prefer to let @MattyMay merge this and setup AG all at once. GH Actions will push to dockerhub after merge.

**Pull request merged**
